src/components/implementation/no_interface/scheddev/sl.h

```diff
@@ -174,10 +192,23 @@ sl_cs_exit(void)
 	if (!ps_cas(&sl__globals()->lock.u.v, cached.v, 0))    goto retry;
 }
 
+static inline microsec_t
+sl_cyc2usec(cycles_t cyc)
+{ return cyc / sl__globals()->cyc_per_usec; }
+
+static inline microsec_t
+sl_usec2cyc(microsec_t usec)
+{ return usec * sl__globals()->cyc_per_usec; }
+
 static inline cycles_t
 sl_now(void)
 { return ps_tsc(); }
 
+static inline microsec_t
+sl_now_usec(void)
+{ return sl_cyc2usec(sl_now()); }
+
```

Comment:
You're saying that `sl_now_usec` existed before in the code?  Why is it not removed in another place in this commit?